source/includes/_api.md

```diff
@@ -337,3 +338,17 @@ var callAttributes = {
 `pc`  | Required | Object | RTCPeerConnection associated with this connection.
 `conferenceID`  | Required | String (256 bytes) | It is generated by the origin server.
 `callAttributes`  | Required | JSON | Contains information about the call.
+
+## callstats.getTurnCredentials()
+
+```javascript
+callstats.getTurnCredentials(appId, appSecret)
+  .then((iceServers) => {
+    var pc_config = {'iceTransports': 'all','iceServers': iceServers};
+    var pc = new RTCPeerConnection(pc_config, {optional: [{DtlsSrtpKeyAgreement: true}]});
+  })
+  .catch((err) => {
+
+  });
```

Comment:
doesnt this API make it very easy to abuse?
anyone could just go to the code of a supported app in browser and extract this line with all needed credentials. this could then be used anywhere while the app has to pay us?

Reply:
@lennart-csio  No, This is api is not accessible by everyone. 

We have a postgress flag for the appID called taasEnabled, This flag has to be true to obtain turnCredentials.

At the moment there is no way to set the flag without contacting us.

Reply:
May be we can "Contact us for enabling turn usage" in the documentation to prevent the cofusion.

Reply:
It does say already "This is an on-demand feature, please contact sales@callstats.io for enabling this feature. "

Reply:
@karthikbr82 yes, i see. however, if there is an app that is taasEnabled and has this integrated, then any user can extract the line from the source code and use it himself without the app knowing

Reply:
@lennart-csio Exactly, Noting can be hidden in javascript. It was possible to do get the same when we were sending the credentials in recomenedConfig callback. Users can also extract turn credentials from authenticate response which we use for PCT.

Reply:
@lennart-csio @karthikbr82 

Aren't the credentials rotating?

Reply:
yes they are